static/script/historian.js

```diff
@@ -121,22 +121,13 @@ define(
              * @returns {Boolean} True if the history stack contains one or more valid return URLs.
```

Comment:
Just need to update the comment regarding "This excludes the 'return to broadcast' special case."